### ISSUE TITLE
Suggest `nth(X)` instead of `skip(X).next()`

### ISSUE THREAD

**Issue by shepmaster** [Manishearth/rust-clippy#1112]
```rust
// Old
.skip(X).next()
// New
.nth(X)
```

**Comment by mcarton** [comment]
I thought we had that but it's slightly different: https://github.com/Manishearth/rust-clippy/issues/1009 so :+1:

**Comment by shepmaster** [comment]
Yeah, I was worried it already existed but it wasn't triggering and I couldn't see it in the list 😇  or any other issues or PRs
